src/index.html

```diff
@@ -15,37 +15,37 @@
 
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
+  <body class="body">
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo">
```

Comment:
refs should always be there, that's why we use links – they refer to something
```suggestion
      <a class="header__logo" href="#">
```